### DISCUSSION TITLE
v6.0.0

### DISCUSSION THREAD

**Discussion by cemrich** [mediathekview/zapp#263] [Announcements]
* Mediathek-Suche erweitert: Filtern nach Sender und Sendungslänge
* Suchvorschläge entfernt
* Sender-Logo in Benachrichtigung angezeigt
* Beim Teilen von Sendungen wird immer die höchste verfügbare Qualität verwendet
* FAQ-Eintrag zu Chromecast hinzugefügt

<hr /><em>This discussion was created from the release <a href='https://github.com/mediathekview/zapp/releases/tag/v6.0.0'>v6.0.0</a>.</em>

**Comment by Tom234** [comment]
Hallo und Danke für die Android App.
Leider kann ich den SEnder ORF nicht finden, sehr wohl aber einige Sendungen daraus, wenn ich danach suche.
Leider kann ich diese Sendungen in Zapp nicht herunterladen.
Gibt es eine Möglichkeit das zu ändern:
o  ORF als Sender darstellen
o  ORF Sendungen Downloadbar?
Danke,
Thomas

**Comment by cemrich** [comment]
Hallo und danke für das Feedback!

Zum Live-Stream: ORF und SRF sind leider außerhalb von Deutschland nicht erreichbar. Da wir mit Zapp primär Nutzer in Deutschland ansprechen, lohnt sich der Aufwand nicht, für diese Sender einen Workaround zu finden.

Zum Download: Die ORF-Sendungen liegen in einem Video-Format vor, das man technisch nicht am Stück (als Datei) herunter laden kann. Man _könnte_ sie offline verfügbar machen - dann allerdings nur innerhalb von Zapp und nicht als Datei-Download (ähnlich wie in Youtube). Das Problem daran: Wir können keine zwei verschiedenen Downloader (einen für ORF und einen für die anderen Sendungen) gleichzeitig pflegen - dafür fehlen schlicht Zeit und Ressourcen. Das heißt, der Dateidownload würde für alle anderen Sender weg fallen - und ich glaube nicht, dass das bei den meisten Nutzern gut ankommen würde.
